modin/pandas/test/utils.py

```diff
@@ -924,6 +920,8 @@ def execute_callable(fn, inplace=False, md_kwargs={}, pd_kwargs={}):
                     assert (
                         pd_e.args == raising_exceptions.args
                     ), f"not acceptable Pandas' exception: [{repr(pd_e)}]"
+                elif raising_exceptions is not False:
```

Comment:
When can we get into this branch if we already have `if raising_exceptions`, and why do we need this branch?

Reply:
This is the only way to disable exception checking, it only works when `raising_exceptions=False`.